recipes/y-py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "y-py" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yjs/y-crdt/archive/v{{ version }}.tar.gz
+  sha256: df7ad800517766c2b73b02e3c07679c04d107f2b12c59dd7d0f6c3513a3918c6
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install ./y-py -vv
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+requirements:
+  build:
+    - {{ compiler('gcc') }}  # [linux]
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - maturin
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - y_py
```

Comment:
it has no deps _today_, but having `pip check` can be very helpful for if/when it _does_ grow some other deps, and is generally good to help keep things running over time.